### PR TITLE
Renaming SiRa to SARA (Smart Alerts Runtime Appliance)

### PR DESCRIPTION
**Pull request by efvink**

### CODE REVIEW

File: src/deployment/SmartSignalsDeployment/createUiDefinition.json [hunk 1]
```diff
@@ -5,11 +5,11 @@
   "parameters": {
     "basics": [
       {
-        "name": "siraPrefix",
+        "name": "saraPrefix",
```
Comment (EranGon) [inline src/deployment/SmartSignalsDeployment/createUiDefinition.json]: >saraPrefix [](start = 17, length = 10)

don't use the abbriviation - smartAlertsAppliancePrefix

File: src/deployment/SmartSignalsDeployment/createUiDefinition.json [hunk 2]
```diff
@@ -5,11 +5,11 @@
...
         "type": "Microsoft.Common.TextBox",
-        "label": "The prefix of The SiRA's name",
+        "label": "The prefix of The SARA's name",
```
Comment (EranGon) [inline src/deployment/SmartSignalsDeployment/createUiDefinition.json]: >SARA [](start = 36, length = 4)

Smart Alerts Applicance

File: src/deployment/SmartSignalsDeployment/DeployManagedApp.ps1 [hunk 3]
```diff
@@ -51,8 +51,8 @@ $nettaDirectGroupId = (Get-AzureRmADGroup -SearchString "netta direct").Id.ToStr
 $ownerID = (Get-AzureRmRoleDefinition -Name Owner).Id
 
 Set-AzureRmManagedApplicationDefinition `
-  -Id "/subscriptions/$subscriptionId/resourceGroups/$resourceGroup/providers/Microsoft.Solutions/applicationDefinitions/SiRA" `
-  -DisplayName "Smart Signals Runtime Application" `
+  -Id "/subscriptions/$subscriptionId/resourceGroups/$resourceGroup/providers/Microsoft.Solutions/applicationDefinitions/SARA" `
+  -DisplayName "Smart Alerts Runtime Appliance" `
   -Description "This application will execute and manage the Smart Signals" `
```
Comment (aviled) [inline src/deployment/SmartSignalsDeployment/DeployManagedApp.ps1]: >Smart Signals [](start = 61, length = 13)

Are we still calling everything smart signal everywhere? should this change to smart alerts?

Comment (efvink) [reply src/deployment/SmartSignalsDeployment/DeployManagedApp.ps1]: Yes, I'm going to change it :)

---
In reply to: [169569044](https://github.com/liadtal/Smart-Alerts/pull/81#discussion_r169569044) [](ancestors = 169569044)

File: src/deployment/SmartSignalsDeployment/createUiDefinition.json [hunk 4]
```diff
@@ -5,11 +5,11 @@
...
     "basics": [
       {
-        "name": "siraPrefix",
+        "name": "smartAlertsAppliancePrefix",
         "type": "Microsoft.Common.TextBox",
-        "label": "The prefix of The SiRA's name",
+        "label": "The prefix of The Smart Alerts Applicance's name",
         "defaultValue": "",
-        "toolTip": "The SiRA and all of its resources will have this prefix",
+        "toolTip": "The SARA and all of its resources will have this prefix",
```
Comment (EranGon) [inline src/deployment/SmartSignalsDeployment/createUiDefinition.json]: >SARA [](start = 24, length = 4)

OMG

File: src/deployment/SmartSignalsDeployment/createUiDefinition.json [hunk 5]
```diff
@@ -41,9 +41,9 @@
       }
     ],
     "outputs": {
-      "siraPrefix": "[basics('siraPrefix')]",
+      "smartAlertsAppliancePrefix": "[basics('smartAlertsAppliancePrefix')]",
       "sendgridApiKey": "[steps('emailSettings').sendgridApiKey]",
-      "applicationResourceName": "[concat(basics('siraPrefix'), 'SiRA')]",
+      "applicationResourceName": "[concat(basics('smartAlertsAppliancePrefix'), 'SARA')]",
```
Comment (EranGon) [inline src/deployment/SmartSignalsDeployment/createUiDefinition.json]: >SARA [](start = 81, length = 4)

SAA

File: src/deployment/SmartSignalsDeployment/DeployManagedApp.ps1 [hunk 6]
```diff
@@ -51,9 +51,9 @@ $nettaDirectGroupId = (Get-AzureRmADGroup -SearchString "netta direct").Id.ToStr
 $ownerID = (Get-AzureRmRoleDefinition -Name Owner).Id
 
 Set-AzureRmManagedApplicationDefinition `
-  -Id "/subscriptions/$subscriptionId/resourceGroups/$resourceGroup/providers/Microsoft.Solutions/applicationDefinitions/SiRA" `
-  -DisplayName "Smart Signals Runtime Application" `
-  -Description "This application will execute and manage the Smart Signals" `
+  -Id "/subscriptions/$subscriptionId/resourceGroups/$resourceGroup/providers/Microsoft.Solutions/applicationDefinitions/SARA" `
+  -DisplayName "Smart Alerts Runtime Appliance" `
```
Comment (EranGon) [inline src/deployment/SmartSignalsDeployment/DeployManagedApp.ps1]: >Runtime [](start = 29, length = 7)

remove

File: src/deployment/SmartSignalsDeployment/DeployManagedApp.ps1 [hunk 7]
```diff
@@ -51,9 +51,9 @@ $nettaDirectGroupId = (Get-AzureRmADGroup -SearchString "netta direct").Id.ToStr
 $ownerID = (Get-AzureRmRoleDefinition -Name Owner).Id
 
 Set-AzureRmManagedApplicationDefinition `
-  -Id "/subscriptions/$subscriptionId/resourceGroups/$resourceGroup/providers/Microsoft.Solutions/applicationDefinitions/SiRA" `
-  -DisplayName "Smart Signals Runtime Application" `
-  -Description "This application will execute and manage the Smart Signals" `
+  -Id "/subscriptions/$subscriptionId/resourceGroups/$resourceGroup/providers/Microsoft.Solutions/applicationDefinitions/SARA" `
```
Comment (EranGon) [inline src/deployment/SmartSignalsDeployment/DeployManagedApp.ps1]: >SARA [](start = 121, length = 4)

SmartAlertsAppliance

File: src/functionApp/SmartSignalsFunctionApp/Authorization/AuthorizationManagementClient.cs [hunk 8]
```diff
@@ -19,7 +19,7 @@ namespace Microsoft.Azure.Monitoring.SmartSignals.FunctionApp.Authorization
     using Polly;
 
     /// <summary>
-    /// This class is responsible to manage authorization for the SiRA
+    /// This class is responsible to manage authorization for the SARA
```
Comment (EranGon) [inline src/functionApp/SmartSignalsFunctionApp/Authorization/AuthorizationManagementClient.cs]: >SARA [](start = 66, length = 4)

Smart Alerts Appliance

File: src/functionApp/SmartSignalsFunctionApp/Authorization/AuthorizationManagementClient.cs [hunk 9]
```diff
@@ -45,11 +45,11 @@ public AuthorizationManagementClient(ITracer tracer, IHttpClientWrapper httpClie
         }
 
         /// <summary>
-        /// Verifying if the HTTP request message is authorized to access the SiRA
+        /// Verifying if the HTTP request message is authorized to access the SARA
```
Comment (EranGon) [inline src/functionApp/SmartSignalsFunctionApp/Authorization/AuthorizationManagementClient.cs]: >SARA [](start = 78, length = 4)

Smart Alerts Appliance

File: src/functionApp/SmartSignalsFunctionApp/Authorization/AuthorizationManagementClient.cs [hunk 10]
```diff
@@ -45,11 +45,11 @@ public AuthorizationManagementClient(ITracer tracer, IHttpClientWrapper httpClie
...
         /// </summary>
         /// <param name="req">The HTTP request message</param>
         /// <param name="cancellationToken">The cancellation token.</param>
-        /// <returns>True if the client is authorized to access the SiRA, false otherwise</returns>
+        /// <returns>True if the client is authorized to access the SARA, false otherwise</returns>
```
Comment (EranGon) [inline src/functionApp/SmartSignalsFunctionApp/Authorization/AuthorizationManagementClient.cs]: >SARA [](start = 68, length = 4)

Smart Alerts Appliance

File: src/functionApp/SmartSignalsFunctionApp/Authorization/AuthorizationManagementClient.cs [hunk 11]
```diff
@@ -94,7 +94,7 @@ public AuthorizationManagementClient(ITracer tracer, IHttpClientWrapper httpClie
                 return false;
             }
 
-            // For now, for every permission action allowed on the resource group we allow accessing the SiRA.
+            // For now, for every permission action allowed on the resource group we allow accessing the SARA.
```
Comment (EranGon) [inline src/functionApp/SmartSignalsFunctionApp/Authorization/AuthorizationManagementClient.cs]: >SARA [](start = 105, length = 4)

Smart Alerts Appliance

File: src/functionApp/SmartSignalsFunctionApp/Authorization/IAuthorizationManagementClient.cs [hunk 12]
```diff
@@ -11,16 +11,16 @@ namespace Microsoft.Azure.Monitoring.SmartSignals.FunctionApp.Authorization
     using System.Threading.Tasks;
 
     /// <summary>
-    /// This interface is responsible to manage authorization for the SiRA
+    /// This interface is responsible to manage authorization for the SARA
```
Comment (EranGon) [inline src/functionApp/SmartSignalsFunctionApp/Authorization/IAuthorizationManagementClient.cs]: >SARA [](start = 70, length = 4)

Smart Alerts Appliance

File: src/functionApp/SmartSignalsFunctionApp/Authorization/IAuthorizationManagementClient.cs [hunk 13]
```diff
@@ -11,16 +11,16 @@ namespace Microsoft.Azure.Monitoring.SmartSignals.FunctionApp.Authorization
...
     /// </summary>
     public interface IAuthorizationManagementClient
     {
         /// <summary>
-        /// Verifying if the HTTP request message is authorized to access the SiRA
+        /// Verifying if the HTTP request message is authorized to access the SARA
```
Comment (EranGon) [inline src/functionApp/SmartSignalsFunctionApp/Authorization/IAuthorizationManagementClient.cs]: >SARA [](start = 78, length = 4)

Smart Alerts Appliance

File: src/functionApp/SmartSignalsFunctionApp/Authorization/IAuthorizationManagementClient.cs [hunk 14]
```diff
@@ -11,16 +11,16 @@ namespace Microsoft.Azure.Monitoring.SmartSignals.FunctionApp.Authorization
...
         /// </summary>
         /// <param name="req">The HTTP request message</param>
         /// <param name="cancellationToken">The cancellation token.</param>
-        /// <returns>True if the request is authorized to access the SiRA, false otherwise</returns>
+        /// <returns>True if the request is authorized to access the SARA, false otherwise</returns>
```
Comment (EranGon) [inline src/functionApp/SmartSignalsFunctionApp/Authorization/IAuthorizationManagementClient.cs]: >SARA [](start = 69, length = 4)

Smart Alerts Appliance

**Comment by efvink** [comment]
was merged to features/efvink/rename_SmartSignalResultItem_and_delete_SmartSignalResult

**Pull request closed without merging**